docs/nodes/logging.md

```diff
@@ -0,0 +1,174 @@
+---
+order: 7
+---
+
+## Logging
+
+Structured logging adds detail to logs and allows the node operator to better identify what they are looking for. Tendermint supports log levels on a global and per module basis. This allows the node operator to see only the information they need and the developer to hone in on specific changes they are working on. 
+
+## Configuring Log Levels
+
+
+There are three log levels, `info`, `debug` and `error`. These can be configured either through the command line via  `tendermint start --log-level ""` or within the `config.toml` file.
```

Comment:
Would be good to explain which kinds of messages are in which (especially the difference between info and debug).